### PR TITLE
Updating policy templates name for consistency

### PR DESCRIPTION
**Pull request by bharatrightscale**
### Description

Updating policy templates name for consistency

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: tools/policy_upload_scripts/support_modules/rs_auth.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def rs_oauth(refresh_token):
         shard = shards[shard_idx]
         shard_idx = shard_idx + 1
         try: 
-            uri = "https://us-{}.rightscale.com/api/oauth2".format(shard)
+            uri = "https://moo-93.test.rightscale.com/api/oauth2".format(shard)
```
Comment (rshade) [inline tools/policy_upload_scripts/support_modules/rs_auth.py]: You slippped in a moo-93.test

File: tools/policy_upload_scripts/policy_bootstrapper.py [hunk 2]
```diff
@@ -54,7 +54,7 @@
 shard = auth_info["shard"]
 
 # Base URLs for below
-cm_base_url = "https://us-{}.rightscale.com".format(shard)
+cm_base_url = "https://moo-93.test.rightscale.com".format(shard)
```
Comment (rshade) [inline tools/policy_upload_scripts/policy_bootstrapper.py]: probably needs to set back

File: tools/policy_upload_scripts/support_modules/rs_auth.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def rs_basic_auth(account_id, username, password):
     while (shard_found == False) and (shard_idx < 3): 
         shard = shards[shard_idx]
         shard_idx = shard_idx + 1
-        uri = "https://us-{}.rightscale.com/api/sessions".format(shard)
+        uri = "https://moo-93.test.rightscale.com/api/sessions".format(shard)
```
Comment (rshade) [inline tools/policy_upload_scripts/support_modules/rs_auth.py]: same here

File: tools/policy_upload_scripts/support_modules/rs_user_info.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import requests,json
 
 def rs_user_email(shard, access_token):
-    cm_base_url = "https://us-{}.rightscale.com".format(shard)
+    cm_base_url = "https://moo-93.test.rightscale.com".format(shard)
```
Comment (rshade) [inline tools/policy_upload_scripts/support_modules/rs_user_info.py]: Same here

File: tools/policy_upload_scripts/support_modules/rs_auth.py (hunk 1 above)
Comment (bharatrightscale) [reply tools/policy_upload_scripts/support_modules/rs_auth.py]: That's strange. I didn't even modify this file.

File: tools/policy_upload_scripts/policy_bootstrapper.py (hunk 2 above)
Comment (bharatrightscale) [reply tools/policy_upload_scripts/policy_bootstrapper.py]: Same as above.

**Pull request merged**